scripts/dts/extract/globals.py

```diff
@@ -167,6 +167,9 @@ def find_node_by_path(nodes, path):
 
 
 def get_reduced(nodes, path):
+    if 'compat_count' not in get_reduced.__dict__:
+        get_reduced.compat_count = {}
+
     # compress nodes list to nodes w/ paths, add interrupt parent
```

Comment:
This comment seems weird at that place now.

Reply:
moved it back up.